siliconcompiler/report/streamlit_report.py

```diff
@@ -41,7 +41,22 @@ def __init__(self, chip, port=None):
             ("server.port", self.__port)
         ]
 
-        self.__config = {"manifest": self.__manifest}
+        # pass in a json object called __comparison_chips
+        # the key is the chip_name and value is the filepath
+        # if another argument is passed
+        self.__comparison_chips = []
+        self.__comparison_chips_names = []
+        if comparison_chips:
+            for chip_object_and_name in comparison_chips:
+                chip_file_path = \
+                    os.path.join(self.__directory,
+                                 f"{chip_object_and_name['name']}.json")
+                self.__comparison_chips.append({'chip': chip_object_and_name['chip'],
+                                                'name': chip_file_path})
+                self.__comparison_chips_names.append(chip_file_path)
```

Comment:
I thought it would be more efficient to do it this way. I would have to write something like `[x['name'] for x in self.__comparison_chips]` on line 59. I thought that since I was already iterating through the list earlier, I might as well do the computation then. I realize this might be such a small difference that it is pointless to include, and I'm not even sure if there is a difference at all. If you prefer this other way, I will change it to that.